test/functional/api/v2/hostgroups_controller_test.rb

```diff
@@ -18,13 +25,43 @@ class Api::V2::HostgroupsControllerTest < ActionController::TestCase
     assert !show_response.empty?
   end
 
+  def last_hostgroup
```

Comment:
Nothing critical, it's just that it struck me as weird that a helper was in the middle of the file, normally I look at the end of the file under private to see these kind of methods